docusaurus.config.js

```diff
@@ -272,24 +272,17 @@ const config = {
           {
             to: "docs",
             position: "left",
-            label: "Getting started",
+            label: "Get started",
             className: "header-text-link",
             target: "_self",
           },
           {
-            to: "docs/learn",
+            to: "learn-wing",
```

Comment:
Can this just be `/learn`?

Reply:
Yeah /learn is already taken, but let me look at this!